### ISSUE TITLE
Any plans to release a sample code for MINI-LM distillation?

### ISSUE THREAD

**Issue by leimao** [microsoft/unilm#105]
Hi,

Thank you for releasing the distilled MINI-LM models from pre-trained Transformer models. I wonder if you have any plans to release a sample code for MINI-LM distillation implementations in either TensorFlow or PyTorch? If people have different pre-trained Transformer models and they want to distill the knowledge, they might be benefited from such scripts. Thank you very much.

Best,

Lei

**Comment by donglixp** [comment]
@WenhuiWang0824

**Comment by VenkatKS** [comment]
Hi, I was just wondering if there is any update regarding this? Having these scripts would be amazingly helpful! 🙂

**Comment by padipadou** [comment]
Do you have any update on this please ?

**Comment by joshdevins** [comment]
Bump. Any update on this would be appreciated. I think there are many researchers and the community at large that would benefit from being able to distill non-BERT LMs.

**Comment by joshdevins** [comment]
@donglixp Any update on this?

**Comment by VenkatKS** [comment]
Yup, I am still interested in these scripts as well, if at all possible to release 🙂

**Comment by luoqishuai** [comment]
+1

**Comment by wasiahmad** [comment]
+1

**Comment by sawwhite** [comment]
+1

**Comment by mrpeerat** [comment]
+1

**Comment by dreamgonfly** [comment]
+1

**Comment by joczu** [comment]
+1

**Comment by xwuShirley** [comment]
It looks like the authors has not yet released the distillation scripts, right?

**Comment by xs1997zju** [comment]
+1
